services/email_verification.js

```diff
@@ -35,45 +41,57 @@ const mailer = require("nodemailer").createTransport({
-        deleteThis: true,
-        id
-    }), async err => {
+    let token = bcrypt.hashSync(`${data.user.email}:${id}`)
+    client.set(`${data.user.email}:email_verify`, token, async err => {
 
-        if (err) send(res, 500, `{code: 500, message: ${err}`);
+        if (err) send(res, 500, `{"code": "500", "message": ${err}`);
 
         // While we have this on urlencoded, the Redis entry serves as a verification point.
         // We want to confirm if this is created by us or maliciously created by someone.
         // However, this is handled by the REST Server, and not this microservice.
+        // TODO : Check if this a Redis entry exist to prevent resends.
 
         else mailer.sendMail({
             to: `${data.user.name} <${data.user.email}>`,
             subject: 'Welcome to Sayonika - Confirm your email!',
             html: `
-              <p align="center">Hey there, we would like to welcome you in Sayonika!</p>
+              <html>
+               <head>
+                 <style>
+                   p {
+                      text-align: center;
+                   } 
+                   a {
+                       text-align: center;
+                   }
+                 </style>
+               </head>
+              <body>
+              <p>Hey there, we would like to welcome you in Sayonika!</p>
               <br>
-              <p align="center">
+              <p>
                 As a security measure we would like to ask you to confirm your account.
                 You will not be able to access your account if you don't confirm your email within 24 hours.
               </p>
               <br>
-              <center><a href="${config.url}/api/v1/verify?token=${id}">Click to confirm your account</a></center>
+              <a href="${config.url}/api/v1/verify?token=${token}">Click to confirm your account</a>
               <br>
-              <p align="center">If you have questions, do not hesitate to ask us on <a href="mailto:hello@sayonika.moe">hello@sayonika.moe</a>.</p>
+              <p >If you have questions, do not hesitate to ask us on <a href="mailto:hello@sayonika.moe">hello@sayonika.moe</a>.</p>
 
-              <p align="left">Happy Modding!</p>
+              <p>Happy Modding!</p>
+              </body>
             `
         }, async(err, info) => {
-             if (err) send(res, 500, `{"code": 500, "message": "failed to send to subject. Reason: ${err}"}`);
-             else send(res, 200, '{"code": 200, "message": "Transport to subject successful"}');
+             if (err) send(res, 500, `{"code": "500", "message": "failed to send to subject. Reason: ${err}"}`);
+             else send(res, 200, '{"code": "200", "message": "Transport to subject successful"}');
              mailer.close();
         });
     });
     // expire token by 24 hours.
-    client.expireat(data.username, parseInt((+new Date)/1000) + 86400);
+    client.expireat(data.user.name, parseInt((+new Date)/1000) + 86400);
```

Comment:
This doesn't match with the key for the `set` above, maybe just remove this and add the TTL argument above as stated before.